lib/gtfsplus/components/GtfsPlusVersionSummary.js

```diff
@@ -190,43 +205,45 @@ export default class GtfsPlusVersionSummary extends Component<Props, State> {
                     <th>Included?</th>
                     <th>Records</th>
                     <th>Validation Issues</th>
-                    <th />
                   </tr>
                 </thead>
                 <tbody>
-                  {getGtfsPlusSpec().map((table, index) => {
-                    const issueCount = this.validationIssueCount(table.id)
-                    const tableLevelIssues = this._getTableLevelIssues(table.id)
-                    return (
-                      <tr
-                        rowSpan={tableLevelIssues ? 2 : 1}
-                        key={index}
-                        className={tableLevelIssues
-                          ? 'danger'
-                          : +issueCount > 0 && 'warning'
-                        }
-                        style={{ color: this.isTableIncluded(table.id) === 'Yes' ? 'black' : 'lightGray' }}>
-                        <td>
-                          {table.name}
-                          {tableLevelIssues
-                            ? <small>
-                              <br />
-                              {tableLevelIssues.length} critical table issue(s):
-                              <ul>
-                                {tableLevelIssues.map((issue, i) =>
-                                  <li key={i}>{issue.description}</li>)}
-                              </ul>
-                            </small>
-                            : null
+                  {getGtfsPlusSpec()
+                    .sort((table1, table2) => table1.name.localeCompare(table2.name))
```

Comment:
Shouldn't we just sort the gtfs.yml file keys instead?

Reply:
Sort that file too I'd say.